### PR TITLE
Completed week 1 portfolio 

### PR DESCRIPTION
**Pull request by yinlow63**
Index: Basic introduction and profile picture with typewriter javascript effect.
Photography: Photography Gallery with pictures link to my Flickr album.
About: A page with longer self-introduction.

Complete with navigation bar and footer of social links.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,50 @@
...
+                            </ul>
+                        </div> <!-- navbar collapse -->
+                    </nav> <!-- navbar -->
+                </div>
+            </div> <!-- row -->
+        </div> <!-- container -->
+    </div> <!-- navigation -->
+
+
+    <!-- <div id="content">
```
Comment (EarlOu) [inline portfolio/src/main/webapp/index.html]: Given we are using git, it's good to just remove the unused code.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,113 @@
-#content {
+/* #content {
```
Comment (EarlOu) [inline portfolio/src/main/webapp/style.css]: same here

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,113 @@
...
+h4,
+h5,
+h6 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  color: #32333c;
+  margin: 0px; }
+
+h1 {
+  font-size: 60px; }
```
Comment (EarlOu) [inline portfolio/src/main/webapp/style.css]: The best practice to set font size is by using relative unit, e.g. %
https://blog.logrocket.com/the-elements-of-responsive-typography/

**Review by EarlOu — CHANGES_REQUESTED** [review]
Coding style is pretty consistent! I also found you added many libraries, I'm not that familiar with them so I can only provide you minimal feedback. Good job for the week 1 work.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (yinlow63) [reply portfolio/src/main/webapp/index.html]: Done! Thanks! I removed all commented unused code.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (yinlow63) [reply portfolio/src/main/webapp/style.css]: Done! Thanks! I removed all commented unused code in css style.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (yinlow63) [reply portfolio/src/main/webapp/style.css]: Done by updating with responsive fonts using relative unit (% and rem).

**Review by EarlOu — APPROVED** [review]

**Pull request merged**